examples/llrp_manager/src/manager.cpp

```diff
@@ -739,6 +763,74 @@ void LLRPManager::SetDeviceLabel(int target_handle, const std::string& label)
   }
 }
 
+void LLRPManager::ResetDevice(int target_handle)
+{
+  auto mgr_pair = managers_.find(active_manager_);
+  if (mgr_pair != managers_.end())
+  {
+    auto target = targets_.find(target_handle);
+    if (target != targets_.end())
+    {
+      RdmCommand cmd_data;
+      RdmResponse resp_data;
+
+      cmd_data.dest_uid = target->second.prot_info.uid;
+      cmd_data.subdevice = 0;
+      cmd_data.command_class = kRdmCCSetCommand;
+      cmd_data.param_id = E120_RESET_DEVICE;
+      cmd_data.datalen = 0;
+
+      if (SendRDMAndGetResponse(mgr_pair->first, target->second.prot_info.cid, cmd_data, resp_data))
+      {
+        target->second.identifying = !target->second.identifying;
```

Comment:
Looks like this line and line 819 are leftover from a copy and paste from IdentifyDevice(), they should be removed.

Reply:
I have removed the lines